spec/rubocop/cop/rails/duplicate_association_spec.rb

```diff
@@ -227,4 +227,15 @@ class Post < ApplicationRecord
       RUBY
     end
   end
+
+  describe 'a class that does not descend from activerecord' do
```

Comment:
That's just my two cents:

```suggestion
  describe 'a class that does not descend from Active Record' do
```